benchmarks/ClusterBenchmark/Program.cs

```diff
@@ -159,10 +160,11 @@ private static IClusterProvider ClusterProvider()
             }
         }
 
-        private static MongoIdentityLookup GetIdentityLookup()
+        private static IIdentityLookup GetIdentityLookup()
         {
             var db = GetMongo();
-            var identity = new MongoIdentityLookup("mycluster", db);
+            var identity = new IdentityStorageLookup(
+                new MongoIdentityStorage("mycluster", db.GetCollection<PidLookupEntity>("pids")));
```

Comment:
Should we make "pids" configurable?

Reply:
This is configurable, since it is passed in by the user

Reply:
doh, it's in the consumer code, my bad